src/components/404.vue

```diff
@@ -1,373 +1,179 @@
-        fill="black"
-      />
-      <path
-        d="M107.257 41.0207H110V33.6459C110 31.613 108.898 30.3154 106.609 30.3154C104.902 30.3154 103.628 31.1372 102.958 31.9374V30.5749H100.215V41.0207H102.958V33.992C103.412 33.3648 104.254 32.7592 105.334 32.7592C106.501 32.7592 107.257 33.2566 107.257 34.7056V41.0207Z"
-        fill="black"
-      />
-    </svg>
-    <div id="logo">
-      <div id="first_logo_box" class="color_box"></div>
-      <div id="second_logo_box" class="color_box"></div>
-      <div id="third_logo_box" class="color_box"></div>
-      <div id="fourth_logo_box" class="color_box"></div>
+<div>
+  <div class = "body-container">
+    <!--TOP BAR-->
+    <div class="container p-0">
+        <div class="row top-bar justify-content-between no-gutters">
+            <div class="col-10">
+                <img class="logo" src="@/assets/images/logo.svg" alt = "logo" />
+              </div>
+        </div>
     </div>
-    <div id="band">
-      <div id="first_four">
-        <div id="first_box1" class="color_box"></div>
-        <div id="second_box1" class="color_box"></div>
-        <div id="third_box1" class="color_box"></div>
-        <div id="fourth_box1" class="color_box"></div>
+    <!-- <img data-v-ef68022e="" src="img/logo.4d084161.svg" alt="logo" class="logo"> -->
+    <div class = "message-container">
+      <img class = "img-404" src="@/assets/images/404.svg" alt = "404">
+      <div class = "oops-wrapper">
+        <div id="oops" class="color_box">
+          Oops...
+          <br/>Page Not Found <img class = "thinking-face" src = "@/assets/images/thinking_face.svg">
+        </div>
       </div>
-      <div id="zero">
-        <div id="first_box2" class="color_box"></div>
-        <div id="second_box2" class="color_box"></div>
-        <div id="third_box2" class="color_box"></div>
-        <div id="fourth_box2" class="color_box"></div>
-      </div>
-      <div id="second_four">
-        <div id="first_box3" class="color_box"></div>
-        <div id="second_box3" class="color_box"></div>
-        <div id="third_box3" class="color_box"></div>
-        <div id="fourth_box3" class="color_box"></div>
-      </div>
-      <div id="oops" class="color_box">
-        Oops...
-        <br />Page Not Found 🤔
-      </div>
-      <div id="back_to_home">Back to home</div>
     </div>
-    <div id="bottom_links">
-      <div id="copyright">Cornell DTI @ 2019</div>
-      <div id="love">Built with LOVE</div>
-      <div id="privacy">Privacy Policy</div>
+    <div class="back_to_home">
+      <a class = "back_to_home_link" href = "/login">Back to home</a>
     </div>
-  </body>
-</html>
+    <div class = "footer-container">
```

Comment:
Since its been a while, can you remind me why you did not end up just importing the Footer component? This works and looks good, but would definitely be better to avoid duplicating code.

Reply:
I remember that when just copying over the footer html + css there were issues with formatting so I had to change some of the css to work on this page. Should I still import it + is there a way to import and overwrite its css?